<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginForm</class>
 <widget class="QWidget" name="loginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="type" stdset="0">
   <string>typeA</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>164</y>
     <width>296</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="login_login">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="type" stdset="0">
       <string>typeA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_register">
      <property name="text">
       <string>Register</string>
      </property>
      <property name="type" stdset="0">
       <string>typeA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>24</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>User name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_username">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>24</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>57</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_password">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>57</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="login_remember">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>127</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>90</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SIM - card PIN:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_pin">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
